index.html

```diff
@@ -112,18 +109,30 @@
 			</select>
 		</form>
 		<br>
-		<label for = "AlchemyMysteriesCheckbox">Alchemy Mysteries DLC</label>
-		<input type = "checkbox" id = "AlchemyMysteriesCheckbox" name = "AlchemyMysteriesCheckbox">
+		<label id = "dlcLabel">DLC:</label>
 		<br>
-		<label for = "ArtOfAdventureCheckbox">Art of Adventure DLC</label>
-		<input type = "checkbox" id = "ArtOfAdventureCheckbox" name = "ArtOfAdventureCheckbox">
-		<div class = "readyButton">
-			<button type = "button">
-				Search
-			</button>
+		<br>
+		<form>
+			<label for = "AlchemyMysteriesCheckbox">Alchemy Mysteries DLC</label>
+			<input type = "checkbox" id = "AlchemyMysteriesCheckbox" name = "AlchemyMysteriesCheckbox">
+		</form>
+		<form>
+			<label for = "ArtOfAdventureCheckbox">Art of Adventure DLC</label>
+			<input type = "checkbox" id = "ArtOfAdventureCheckbox" name = "ArtOfAdventureCheckbox">
+		</form>
+		<br>
+		<div class = "languageDropdown">
```

Comment:
Move the language dropdown back to the top. It should be one of the first things the user sees and easily accessible from when the page loads.